src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt

```diff
@@ -52,15 +52,15 @@ class CrashModule(
 
             if (key == null) {
                 if (anyModded) {
-                    addComment(CommentOptions(moddedMessage)).toFailedModuleEither().bind()
-                    resolveAsInvalid().toFailedModuleEither().bind()
+                    addComment(CommentOptions(moddedMessage))
```

Comment:
add comment should be always after resolving, and it should only happen if the module didnt fail to do the resolve